manifests/opsmanager.pp

```diff
@@ -0,0 +1,59 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::params::opsmanager_user,
+  String[1] $group                               = $mongodb::params::opsmanager_group,
+  Variant[Boolean, String[1]] $ensure            = $mongodb::params::opsmanager_ensure,
+  String[1] $package_name                        = $mongodb::params::opsmanager_package_name,
+  Boolean $package_ensure                        = $mongodb::params::opsmanager_package_ensure,
+  Boolean $service_enable                        = $mongodb::params::opsmanager_service_enable,
+  Boolean $service_manage                        = $mongodb::params::opsmanager_service_manage,
+  String[1] $service_name                        = $mongodb::params::opsmanager_service_name,
+  String[1] $download_url                        = $mongodb::params::opsmanager_download_url,
+  String[1] $mongo_uri                           = $mongodb::params::opsmanager_mongo_uri,
+  String[1] $hostname                            = $mongodb::params::opsmanager_hostname,
+  String[1] $protocol                            = $mongodb::params::opsmanager_protocol,
+  Stdlib::Port $port                             = $mongodb::params::opsmanager_port,
+  String[1] $opsmanager_url                      = $mongodb::params::opsmanager_url,
```

Comment:
Should this be `Stdlib::Httpurl`?

Reply:
My only concern is that Httpurl is specific to non-ssl URLs, just as Httpsurl is specific to ssl urls. Currently the url can be replaced by either 'https://blahblah' or 'http://blahblah' and it behaves the same way.

Reply:
Oops! Clearly I misread/misunderstood the docs. Thanks for the clarification.